src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,28 @@ title = "Releases"
 weight = 30
 +++
 
+## v0.8.0
+
+* Added support for connecting clusters that use the OVNKubernetes CNI plugin.
```

Comment:
@tpantelis, I discussed this with @mangelajo earlier today, and we both think it's a good idea that with 0.8.0 OVN is only supported with non-Globalnet deployments.

Reply:
Done